common/changes/@azure-rest/agrifood-farming/core-move-typeguard-utils-into-core-util_2022-08-10-21-54.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I'm not sure about  these changes under common/*. Is it possible to remove these changes?

Reply:
Done!

Reply:
This looks like the `rush change` feature that we don't use (but the CADL team does, which is the only reason I know about it.)